testsuite/MDAnalysisTests/core/test_universe.py

```diff
@@ -498,6 +498,15 @@ def test_slicing_step_with_start_stop():
         assert_equal(universe.trajectory.timeseries(universe.atoms).shape,
                      (3341, 10, 3),
                      err_msg="Unexpected shape of trajectory timeseries")
+    @staticmethod
+    def test_slicing_step_dt():
+        universe = MDAnalysis.Universe(PDB_small, DCD)
+        times=[ts.time for ts in universe.trajectory]
```

Comment:
Please follow [PEP 8](https://www.python.org/dev/peps/pep-0008/) and use spaces around operators.

(Applies to this line and the following ones, too.)